docs/gds-supported-platform/aws-service-operator.md

```diff
@@ -0,0 +1,27 @@
+# Using AWS Service Operator
+
+AWS Service Operator is a tool we use to allow GSP users to write kubeyaml resources that will directly map to AWS resources.
+It consists of a container that runs essentially a daemon, and a (slightly modified in our case) kubeyaml config that sets up the container, provides a bunch of custom resource definitions (e.g., there is a definition in there for SQS Queues), etc. - it also gives the container access to monitor stuff going on around the cluster.
+The daemon monitors the k8s cluster for such custom resources being created and will deploy to AWS the relevant CloudFormation to create the requested resource.
+The end result of this is that developers can create certain types of AWS resources directly inside their kubeyaml. Here's an example:
+```yaml
+
+- kind: SQSQueue
+  apiVersion: service-operator.aws/v1alpha1
+  metadata:
+    name: alexs-test-queue
+  spec:
+    messageRetentionPeriod: 3600
+    maximumMessageSize: 1024
+```
+This will create an SQS Queue on AWS named alexs-test-queue, with a message retention period of 1 hour, and a maximum message size of 1KiB.
```

Comment:
Once you've created the queue, how do you connect to it?

Reply:
Looks like we need to do #302 for that to work well.